client/state/posts/actions/save-post.js

```diff
@@ -34,14 +34,7 @@ export function savePost( siteId, postId = null, post, silent = false ) {
 				dispatch( savePostSuccess( siteId, postId, savedPost, post, silent ) );
 				dispatch( receivePost( savedPost ) );
 			},
-			( error ) => {
-				dispatch( {
-					type: POST_SAVE_FAILURE,
-					siteId,
-					postId,
-					error,
-				} );
-			}
+			() => {}
```

Comment:
Should we just remove this one?

Reply:
Omitting is not quite the same, because then it becomes an unhandled exception.

Maybe that's how it _should_ be, but this keeps consistency with how it effectively worked previously.

Reply:
Got it 👍